### PR TITLE
add artist portfolio repeatable field

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

**Comment by mixcas** [comment]
This needs styling

File: src/components/artistas/ArtistaForm.jsx [hunk 1]
```diff
@@ -66,7 +69,18 @@ class ArtistaForm extends Component {
   }
 
   addArtista() {
-    const { name, active, country, gallery, galleryUrl, bioRawContent, cvRawContent, images, video } = this.state;
+    const {
```
Comment (varffs) [inline src/components/artistas/ArtistaForm.jsx:72]: this is def a better format for these things

File: src/components/artistas/ArtistaForm.jsx [hunk 2]
```diff
@@ -345,6 +380,17 @@ class ArtistaForm extends Component {
           </div>
         </div>
 
+        <ArtistaPortfolio
+          enabled={this.state.loading}
+          items={this.state.portfolio}
+          handlePortfolioChange={this.handlePortfolioChange}
+        />
+
+        <div className='grid-row margin-bottom-basic'>
```
Comment (varffs) [inline src/components/artistas/ArtistaForm.jsx]: what is this?

File: src/components/artistas/ArtistaPortfolio.jsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import { toastr } from 'react-redux-toastr';
+
+import Uploads from '../fields/Uploads';
+import { ToastrOptionsConfirm } from '../../utilities/toastr.js';
```
Comment (varffs) [inline src/components/artistas/ArtistaPortfolio.jsx:5]: we should use this utility more as a good overlay

Comment (varffs) [reply src/components/artistas/ArtistaPortfolio.jsx:5]: I think we have basic js confirms somewhere that should be replaced

**Comment by mixcas** [comment]
@varffs rest of the code is fine?

File: src/components/catalogos/CatalogosListItem.jsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ import { ToastrOptionsConfirm, ToastrOptionsSuccess } from '../../utilities/toas
 
 const CatalogosListItem = ({ catalogo, firebase }) => {
   const { key } = catalogo;
-  const { title, year, medium } = catalogo.value;
+  const { title, medium } = catalogo.value;
```
Comment (varffs) [inline src/components/catalogos/CatalogosListItem.jsx:12]: this looks like a missed edit for different work?

File: src/components/artistas/ArtistaPortfolio.jsx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+class ArtistaPortfolio extends Component {
+  state = {
```
Comment (varffs) [inline src/components/artistas/ArtistaPortfolio.jsx:8]: this is just being used to manage the values in the add item form right?

**Review by varffs — COMMENTED** [review]
questions but ready to merge

File: src/components/artistas/ArtistaForm.jsx (hunk 1 above)
Comment (mixcas) [reply src/components/artistas/ArtistaForm.jsx:72]: ikr?! its better when it's more than 3

File: src/components/artistas/ArtistaPortfolio.jsx (hunk 3 above)
Comment (mixcas) [reply src/components/artistas/ArtistaPortfolio.jsx:5]: I think they all got changed already :)

File: src/components/artistas/ArtistaPortfolio.jsx (hunk 5 above)
Comment (mixcas) [reply src/components/artistas/ArtistaPortfolio.jsx:8]: Yes, the component state

**Pull request merged**